redisinsight/api/src/modules/database-recommendation/database-recommendation.controller.ts

```diff
@@ -76,4 +80,30 @@ export class DatabaseRecommendationController {
   ): Promise<DatabaseRecommendation> {
     return await this.service.update(clientMetadata, id, dto);
   }
+
+  @Delete('')
+  @ApiEndpoint({
+    statusCode: 200,
+    description: 'Delete many recommendations by ids',
+    responses: [
+      {
+        status: 200,
+        description: 'Delete many recommendations by ids response',
+        type: DeleteDatabaseRecommendationDto,
+      },
+    ],
+  })
+  @UsePipes(
+    new ValidationPipe({
+      transform: true,
+      whitelist: true,
+      forbidNonWhitelisted: true,
+    }),
+  )
+  async bulkDeleteDatabaseRecommendation(
+    @BrowserClientMetadata() clientMetadata: ClientMetadata,
```

Comment:
`BrowserClientMetadata` related to browser endpoints, we should use own decorator here. Anyway since I missed this even in previous PRs and currently these endpoint database param matches to browser we can go with it